src/ray/protobuf/experimental/autoscaler.proto

```diff
@@ -17,7 +17,101 @@ option cc_enable_arenas = true;
 
 package ray.rpc.autoscaler;
 
-import "src/ray/protobuf/experimental/instance_manager.proto";
+// ============= Instance Definition ====================
+enum RayNodeKind {
+  UNKNOWN = 0;
+  HEAD = 1;
+  WORKER = 2;
+}
+// A node type that's available for the cluster.
+message InstanceType {
+  // the name of the instance type. e.g. "m4.large"
+  string type_name = 1;
+  RayNodeKind node_kind = 2;
+  // avaialble resources on the node. such as {"CPU": 4, "GPU": 1}
+  map<string, double> resources = 3;
+}
+
+// Represents the state of a launched instance.
+// An instance is considered launched as long as
+// it has a unique instance_id associated with it.
+//
+// Note a launched instance may be DEAD. In this case,
+// the state will be garbage collected after some timeout
+// period (by default 30 minutes).
+message Instance {
+  enum InstanceStatus {
```

Comment:
Other than autoscaler, does gcs health check update InstanceStatus?

Reply:
No - this is purely from autoscaler's POV.